### PR TITLE
Update sbt-scalariform

### PR DESCRIPTION
**Pull request by gmethvin**
This creates a pretty large set of changes but I'm okay with most of the formatting changes introduced here. Perhaps we can tweak the settings to reduce the impact.

### CODE REVIEW

File: framework/project/plugins.sbt [hunk 1]
```diff
@@ -16,18 +16,16 @@ logLevel := Level.Warn
 
 scalacOptions ++= Seq("-deprecation", "-language:_")
 
+resolvers += Resolver.typesafeRepo("releases")
```
Comment (benmccann) [inline framework/project/plugins.sbt]: I would guess you don't need this line

Comment (marcospereira) [inline framework/project/plugins.sbt]: It is not. ;-)

**Comment by marcospereira** [comment]
@gmethvin, a possible tweak is the following:

```scala
  val playScalariformSettings = scalariformSettings ++ Seq(
    ScalariformKeys.preferences := ScalariformKeys.preferences.value
      .setPreference(DanglingCloseParenthesis, Preserve)
      .setPreference(SpacesAroundMultiImports, true)
  )
```
And then:
```scala
def playCommonSettings: Seq[Setting[_]] = playScalariformSettings ++ Seq( ...
```
This will reduce the change set significantly, but since the change set is already pretty large, I don't see the point to add specific configuration.

**Comment by marcospereira** [comment]
By the way, the build is failing.

**Comment by marcospereira** [comment]
@gmethvin this needs a rebase now.

**Comment by gmethvin** [comment]
Going to resubmit the PR since I need to do the re-format anyway...

**Pull request closed without merging**